extensions/infinispan-client/deployment/src/main/java/io/quarkus/infinispan/client/deployment/devservices/InfinispanDevServicesConfig.java

```diff
@@ -54,6 +56,14 @@
     @ConfigItem(defaultValue = "infinispan")
     public String serviceName;
 
+    /**
+     * List of the artifacts to download and be present in the server lib. For example a maven coordinate
+     * (org.postgresql:postgresql:42.3.1) or dependency location url.
```

Comment:
I personally don't understand what  this does (and I assume users won't either), so maybe expand on this a little more?

Furthermore, what happens if the string is invalid? What kind of error does Infinispan display?

Reply:
The container won't start if you ask for an invalid maven dependency. This is useful when using persistence with caches IllegalArgumentException: Artifact 'org.postgresql:postgresql:jjj' not found

Reply:
Sure, but I think the javadoc should reflect this

Reply:
I made an adjustment and force pushed in an attempt to get this in 2.6.1.Final.